client/extensions/woocommerce/state/sites/setup-choices/actions.js

```diff
@@ -9,6 +9,8 @@ import { setError } from '../status/wc-api/actions';
 import {
 	WOOCOMMERCE_SETUP_CHOICES_REQUEST,
```

Comment:
WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST should be above: WOOCOMMERCE_SETUP_CHOICES_REQUEST